### ISSUE TITLE
using great_tables in shiny applications

### ISSUE THREAD

**Issue by isabelizimm** [posit-dev/great-tables#56]
## Prework

- [X] Read and abide by the **great_tables** [code of conduct](https://www.contributor-covenant.org/version/2/1/code_of_conduct/) and [contributing guidelines](https://github.com/posit-dev/great_tables/blob/main/.github/CONTRIBUTING.md).
- [X] Search for duplicates among the [existing issues](https://github.com/posit-dev/great_tables/issues) (both open and closed).

## Proposal

Hello there 👋 

I was trying to use `great_tables` in conjunction with `shiny`, and ended up figuring out a combination that works, something like `ui.HTML(gt.GT(df)._repr_html())`. However, it would be awesome if Shiny was a little smarter about handling GT objects. 

Here's a reprex, using great-tables==0.1.0 and shiny==0.6.0.
```python
from shiny import App, render, ui
import great_tables as gt

app_ui = ui.page_fluid(
    ui.output_ui("txt"),
)


def server(input, output, session):
    @output
    @render.ui
    def txt():
        return ui.HTML(gt.GT(gt.exibble)._repr_html_())

app = App(app_ui, server)
```

**Comment by gshotwell** [comment]
Awesoje, FWIW you no longer need `@output` in shiny

**Comment by machow** [comment]
Released in v0.1.2 (see #59). Thanks for the heads up on output, seems like a handy simplification!
